### PR TITLE
Do not display private props and methods

### PR DESCRIPTION
**Pull request by KnorpelSenf**
We should not list private things in the ref

### CODE REVIEW

**Comment by KnorpelSenf** [comment]
It also adds missing docs and link to source for constructors

File: site/api/components/Class/Constructors.tsx [hunk 1]
```diff
@@ -13,13 +15,15 @@ export function Constructors({
...
   }
-  const items = ctors.map(({ accessibility, name, params }, i) => (
+  return ctors.map((v) => (
     <>
-      {accessibility ? `${accessibility} ` : undefined}
-      <span style="color: rgb(98, 232, 132);">{name}</span>(
-      <Params getLink={getLink}>{params}</Params>);
-      {"\n"}
+      <CodeBlock>
+        {v.accessibility ? `${v.accessibility} ` : undefined}
```
Comment (roj1512) [inline site/api/components/Class/Constructors.tsx:21]: v.accessibility should be inside StyleKw

Comment (KnorpelSenf) [reply site/api/components/Class/Constructors.tsx:21]: Right

File: site/api/components/Class/Constructors.tsx [hunk 2]
```diff
@@ -13,13 +16,15 @@ export function Constructors({
...
   }
-  const items = ctors.map(({ accessibility, name, params }, i) => (
+  return ctors.map((v) => (
     <>
-      {accessibility ? `${accessibility} ` : undefined}
-      <span style="color: rgb(98, 232, 132);">{name}</span>(
-      <Params getLink={getLink}>{params}</Params>);
-      {"\n"}
+      <CodeBlock>
+        {v.accessibility ? <StyleKw>`${v.accessibility} `</StyleKw> : undefined}
```
Comment (roj1512) [inline site/api/components/Class/Constructors.tsx:22]:
```suggestion
        {v.accessibility ? <StyleKw>{v.accessibility}{" "}</StyleKw> : undefined}
```

Comment (KnorpelSenf) [reply site/api/components/Class/Constructors.tsx:22]: That would still style the space with the wrong colour … is there anything else that this improves?

Comment (roj1512) [reply site/api/components/Class/Constructors.tsx:22]: @KnorpelSenf No one sees the color of the spaces. You should fix it all other places if you are able to see the color of the spaces.

**Comment by KnorpelSenf** [comment]
@roj1512 please go ahead and merge this if you're okay with it

**Pull request merged**